safe_control_gym/envs/gym_pybullet_drones/quadrotor.py

```diff
@@ -487,8 +487,10 @@ def _set_action_space(self):
                                            dtype=np.float32)
         else:
             # direct thrust control 
-            self.action_space = spaces.Box(low=np.zeros(action_dim), 
-                                           high=self.MAX_THRUST * np.ones(action_dim), 
+            a_low = self.KF * action_dim * (self.PWM2RPM_SCALE * self.MIN_PWM + self.PWM2RPM_CONST)**2
```

Comment:
@JakobThumm on a second look, I am not sure that multiplying by `action_dim` is entirely correct: it would work well for the 2D quadrotor but, in the 1D case we are essentially commanding all 4 motors at once, so we have a theoretical 4x max thrust (the way you wrote it would be multiplied by 1) and for the 3D quadrotor we will want to take 1x max thrust you compute on each motor (but multiplying by `action_dim` would make it 4 times larger)

Reply:
Note that the previous, looser, possibly unachievable, upper bound was using a `self.MAX_THRUST` compute for all 4 motors collectively